ipn/ipnlocal/local.go

```diff
@@ -730,7 +730,9 @@ func (b *LocalBackend) Shutdown() {
 	b.webClientShutdown()
 
 	if b.sockstatLogger != nil {
-		b.sockstatLogger.Shutdown()
+		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
you could do this in parallel in the block above on line 703

Reply:
That stuff is handling ephemeral nodes, and also I did not want to restructure this too much since I don't fully understand if order of shutdown is important at all.